api/src/test/java/org/pmiops/workbench/workspaces/WorkspaceServiceTest.java

```diff
@@ -127,18 +133,21 @@ DbUser user() {
     }
   }
 
+  @MockBean private AccessTierService accessTierService;
   @MockBean private BillingProjectAuditor mockBillingProjectAuditor;
   @MockBean private Clock mockClock;
+  @MockBean private CloudBillingClient mockCloudBillingClient;
+  @MockBean private FeaturedWorkspaceDao mockFeaturedWorkspaceDao;
   @MockBean private FeaturedWorkspaceService mockFeaturedWorkspaceService;
   @MockBean private FireCloudService mockFireCloudService;
-  @MockBean private CloudBillingClient mockCloudBillingClient;
-  @Autowired private WorkspaceDao workspaceDao;
-  @Autowired private WorkspaceService workspaceService;
-  @MockBean private AccessTierService accessTierService;
+  @MockBean private MailService mockMailService;
+  @MockBean private WorkspaceAuthService mockWorkspaceAuthService;
+  @MockBean UserService userService;
```

Comment:
Rearranging  some stuff

Reply:
great, thanks for the cleanup!

This one should probably be private too, I think.  Also it would be helpful to name it "mock" to clarify its behavior.  `accessTierService` too.